src/components/signup.vue

```diff
@@ -0,0 +1,125 @@
+<template>
+  <q-form
+      @submit="onSubmit"
+      @reset="onReset"
+      class="q-gutter-md"
+    >
+      <q-input
+        rounded filled
+        v-model="name"
+        label="First name *"
+        lazy-rules
+        :rules="[ val => val && val.length > 0 || 'Please type your Valid First Name']"
+      ></q-input>
+      <q-input
+        rounded filled
+        v-model="last_name"
```

Comment:
@sanchitno1 always camelCase please change from last_name to lastName cc @priyankashrma